client/webserver/site/src/css/components.scss

```diff
@@ -0,0 +1,218 @@
+// Reusable components and styles 
+@import "~bootstrap/scss/mixins";
+@import "mixins";
+
+
+button {
+  background-color: transparent;
+  border: 1px solid var(--btn-border-color);
+  border-radius: 5px;
+  outline: none;
+  padding: 0.375rem 0.75rem;
+  user-select: none;
+  font-size: 1rem;
+  color: var(--text-color);
+  white-space: nowrap;
+
+  &.selected {
+    background-color: #38cfb0;
+    border: none;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:hover {
+    text-decoration: none;
+    border-color: var(--btn-hover-border-color);
+    background-color: #7772;
+  }
+
+  &.small {
+    padding: 0.25rem 0.5rem;
+    font-size: .8rem;
+  }
+
+  &.large {
+    padding: 0.5rem 1rem;
+    font-size: 1.25rem;
+  }
+
+  &.feature {
+    background-color: var(--btn-feature-bg);
+    border-color: var(--btn-feature-border-color);
+
+    &:hover {
+      background-color: var(--btn-feature-hover-bg);
+      border-color: var(--btn-featur-hover-border-color);
+    }
+  }
+
+  &.danger {
+    border-color: var(--btn-danger);
+    color: var(--btn-danger);
+
+    &:hover {
+      background-color: var(--btn-danger);
+      color: var(--btn-danger-hover-color);
+    }
+  }
+
+  &.go {
+    background-color: var(--btn-go-bg);
+    color: var(--btn-go-color);
+
+    &:hover {
+      background-color: var(--btn-go-hover-bg);
+    }
+  }
+}
+
+// tables
+.table {
```

Comment:
The goal is to define all table styles right here. This shouldn't be a class. Define a base style for all `<table>` elements, and then nest any variations that we need. All table styles should be defined right here.  The goal should be do delete all other table styling definitions that are sprinkled throughout.